### DISCUSSION TITLE
post/30-days-naudio-docs

### DISCUSSION THREAD

**Discussion by giscus** [markheath/soundcode-blog-comments#80] [Announcements]
# post/30-days-naudio-docs

Mark Heath's Development Blog

https://markheath.net/post/30-days-naudio-docs

**Comment by ahmedhasib24** [comment]
Hi Mark! I am relatively new to NAudio. I am making a game in Unity where I need to capture the microphone input and send it over to the network. I tried the 1.9.0 version for .Net Standard 2.0. But I could not find WaveIn there. I also tried the 2.2.1 version also. I can read the number of micrphones there but I can not select and capture from the specific microphone. And also the WaveIn never stops recording there. But with WaveInEvent I can record but can not select the microphone. I tried all of these by following your demos. But could not make my project functioning properly. May be I am missing something. Can you please guide me how I can use the NAudio for my unity project and how can I get access of all the feature NAudio is offering? Btw, my API  compatibility level is .Net Standard 2.1. Thanks.
